docs/source/developer/repomap.rst

```diff
@@ -255,7 +255,8 @@ Misc Support
 - :ghfile:`numba/core/caching.py` - Disk cache for compiled functions
 - :ghfile:`numba/np/npdatetime.py` - Helper functions for implementing NumPy
   datetime64 support
-
```

Comment:
Does a blank line need adding back in?

Reply:
It doesn't need two lines. I checked the rendered doc.